src/functions-and-arrays.js

```diff
@@ -52,19 +98,47 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) { 
+  let unique=[];
+  for (let i=0; i<wordsUnique.length; i++){
+    if(unique.indexOf(wordsUnique[i])===-1){
+     unique.push(wordsUnique[i]);
+    }
+  }
+   return unique; 
+  }
+  console.log(uniquifyArray(wordsUnique));
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, word) {
+  let result=0;
+  let result2=0;
+ for (let i=0; i<arr.length; i++){
```

Comment:
You aren't using `result` after you declare it, so it could be good to clean up your code after you finished it